app/components/work_packages/activities_tab/journals/item_component/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<%=
+  component_wrapper(class: "work-packages-activities-tab-journals-item-component-edit") do
+    render(Primer::Box.new(my: 3)) do
+      primer_form_with(
+        id: "work-package-journal-form",
+        model: journal,
+        method: :put,
+        data: { turbo: true, turbo_stream: true },
+        url: work_package_activity_path(work_package_id: work_package.id, id: journal.id, filter:),
+      ) do |f|
+        flex_layout do |form_container|
+          form_container.with_row do
+            render(WorkPackages::ActivitiesTab::Journals::NotesForm.new(f))
+          end
+          form_container.with_row(flex_layout: true, mt: 3, justify_content: :flex_end) do |submit_container|
+            submit_container.with_column(mr: 2) do
+              render(Primer::Beta::Button.new(
+                     scheme: :secondary,
+                     size: :medium,
+                     tag: :a,
+                     href: cancel_edit_work_package_activity_path(work_package.id, id: journal.id, filter:),
+                     data: { "turbo-stream": true }
```

Comment:
Nitpick: You're mixing `data: { turbo_stream: true } ` and `data: { "turbo-stream": true }`. I'd prefer to use turbo_stream: true